### PR TITLE
[containerapp] 'az containerapp job create': Fix problem of parsing parameters minExecutions and maxExecutions from --yaml

### PR DESCRIPTION
**Pull request by anandanthony**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by Greedygre** [comment]
Recommend title:

'az containerapp job create': fix problem of parsing parameters minExecutions and maxExecutions from --yaml

**Comment by Greedygre** [comment]
Please fix the conflict.

File: src/containerapp/azext_containerapp/_sdk_models.py [hunk 1]
```diff
@@ -7337,44 +7337,44 @@ def __init__(
 class JobScale(_serialization.Model):
     """Container Apps Job scaling configurations.
 
-    :ivar min_replicas: Optional. Minimum number of container replicas.
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/_sdk_models.py:7340]: I think this should add in azure-cli too.

File: src/containerapp/azext_containerapp/_sdk_models.py [hunk 2]
```diff
@@ -7337,44 +7337,44 @@ def __init__(
...
-    :vartype min_replicas: int
-    :ivar max_replicas: Optional. Maximum number of container replicas. Defaults to 10 if not set.
-    :vartype max_replicas: int
+    :ivar min_executions: Optional. Minimum number of container replicas.
+    :vartype min_executions: int
+    :ivar max_executions: Optional. Maximum number of container replicas. Defaults to 10 if not set.
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/_sdk_models.py:7342]: The description please consistent with [azure-rest-api-specs/specification/app/resource-manager/Microsoft.App/stable/2023-05-01/Jobs.json at 11bbc2b1df2e915a2227a6a1a48a27b9e67c3311 · Azure/azure-rest-api-specs (github.com)](https://github.com/Azure/azure-rest-api-specs/blob/11bbc2b1df2e915a2227a6a1a48a27b9e67c3311/specification/app/resource-manager/Microsoft.App/stable/2023-05-01/Jobs.json#L732)

**Review by Greedygre — APPROVED** [review]

**Comment by Greedygre** [comment]
Hi @zhoxing-ms @yanzhudd 
Could you please help review and merge this PR? Thanks.

**Review by wangzelin007 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**